odoo/sql_db.py

```diff
@@ -216,9 +216,9 @@ def execute(self, query, params=None, log_exceptions=None):
         try:
             params = params or None
             res = self._obj.execute(query, params)
-        except Exception:
+        except Exception as e:
             if self._default_log_exceptions if log_exceptions is None else log_exceptions:
-                _logger.info("bad query: %s", self._obj.query or query)
+                _logger.info("bad query: %s \nERROR: %s", self._obj.query or query, e)
```

Comment:
It would display the traceback, which is not very useful because it would display the traceback when we're logging the error. (Dixit Mr Propre)